packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-image/partner-preset-price/index.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+import ovhManagerCatalogPrice from '@ovh-ux/manager-catalog-price';
+
+import component from './partner-preset-price.component';
+
+const moduleName = 'ovhManagerPciProjectAppsAppImagePartnerPresetPrice';
+
+angular
+  .module(moduleName, [
+    'oui',
+    'pascalprecht.translate',
+    'ngOvhCloudUniverseComponents',
+    ovhManagerCatalogPrice,
+  ])
+  .component('partnerPresetPrice', component)
```

Comment:
**nitpick:** Prefix the component with `pci-project` to avoid component name clash
```suggestion
  .component('pciProjectPartnerPresetPrice', component)
```

Reply:
Done here: https://github.com/ovh/manager/pull/9413/commits/975892ea8b6d0b573b69a30322799c0f84841551